makefiles/e2e.mk

```diff
@@ -85,7 +85,7 @@ e2e-rollout-test:
 
 .PHONY: e2e-multicluster-test
 e2e-multicluster-test:
-	go test -v -coverpkg=./... -coverprofile=/tmp/e2e_multicluster_test.out ./test/e2e-multicluster-test
+	go test -v -coverpkg=./... -timeout=20m -coverprofile=/tmp/e2e_multicluster_test.out ./test/e2e-multicluster-test
```

Comment:
why we need this time longer

Reply:
When bumping the kind version, some test become flaky. That's why I make some wait time for `Eventually`  longer. All time add up and sometimes it'll fail for timeout since the default timeout for `go test` is 10min. Then I add a longer timeout.

Reply:
Should we upgrade kind if our tests will be affected?

Reply:
Upgrade kind for setting up cluster >=1.22. Kind 0.7.0 will fail on that.